README.md

```diff
@@ -37,31 +51,88 @@ alongside the platform. To install your app, upload the manifest to the local
+### Core Components
 
-The graphical SDK will allow you to quickly build and test simple databox apps. To start the sdk run:
+* [Databox container manager](https://github.com/me-box/core-container-manager)
 
-	./databox-start sdk
+* [databox-arbiter](https://github.com/me-box/core-arbiter)
 
-The SDK web UI is available at http://127.0.0.1:8086
+* [databox-export-service](https://github.com/me-box/core-export-service)
 
-To stop the SDK run: 
+* [databox-store-json](https://github.com/me-box/store-json)
 
-	./databox-stop sdk
+* [databox-store-timeseries](https://github.com/me-box/store-timeseries)
 
+* [databox-app-server](https://github.com/me-box/platform-app-server) Server for storing and serving databox manifests
 
-### Develop system components and custom drivers
+### User Components
+#### Drivers
+* [driver-sensingkit](https://github.com/me-box/driver-sensingkit)
+* [driver-google-takeout](https://github.com/me-box/driver-google-takeout)
+* [driver-phillips-hue](https://github.com/me-box/driver-phillips-hue)
+* [driver-os-monitor](https://github.com/me-box/driver-os-monitor)
+* [driver-twitter](https://github.com/me-box/driver-twitter)
+* [driver-tplink-smart-plug](https://github.com/me-box/driver-tplink-smart-plug)    
+#### Apps 
+* [app-light-graph] https://github.com/me-box/app-light-graph
+* [app-twitter-sentiment] https://github.com/me-box/app-twitter-sentiment
+* [app-os-monitor] https://github.com/me-box/app-os-monitor
+                
+### Libraries for writing drivers and apps
+* [lib-node-databox](https://github.com/me-box/node-databox) nodejs lib for databox apps and drivers
+* [lib-python-databox](https://github.com/me-box/lib-python-databox) python lib for databox apps and drivers
+* [lib-go-databox](https://github.com/me-box/lib-go-databox) go lib for databox apps and drivers
+#### API and System specifications
+Databox System Design document can be find [here](./documents/system_overview.md) and General API specifications are [here](./documents/api_specification.md).
+
+### Running the tests
+
+```
+terminal$ ./databox-test 
+
+```
+For more details, have a look [here](./documents/testing.md).
+
+## Contributing
 
-To develop on the platform and core components you can run 
+The databox project welcomes contributions via pull requests see [CONTRIBUTING.md](./documents/CONTRIBUTING.md) for more information. Good start is from having a look on  the current [issues](https://github.com/me-box/databox/issues) and [forking](https://github.com/me-box/databox#fork-destination-box) the databox repo and fixing bugs/issues and submitting a pull request. Read more on Fork and Pull [here](https://help.github.com/articles/fork-a-repo/).
+
+## Versioning
+
+This documentation is up-to-date till this [commit](https://github.com/me-box/databox/tree/a62ed323d98c0a6fd32f020eca9352f8da687c09). The master branches on all components points to the current release and are tagged in git using [semver](http://semver.org/).
```

Comment:
I think we need a conversation about exactly what we do here. I'll add to agenda for Tuesday call.